src/material-experimental/mdc-helpers/_mdc-helpers.scss

```diff
@@ -217,39 +217,55 @@ $mat-typography-level-mappings: (
 ///
 /// @example
 ///   .my-app {
-///     @include mat-mdc-strong-focus-indicators();
+///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
-@mixin mat-mdc-strong-focus-indicators() {
-  // Base styles for the focus indicators.
+@mixin mat-mdc-strong-focus-indicators($config: ()) {
+  // Default focus indicator config.
+  $default-config: (
+    border-style: solid,
+    border-width: 3px,
+    border-radius: 4px,
```

Comment:
If we're only configuring these properties, would it make more sense for them to be passed in as separate parameters?

Reply:
I'm easy either way, @jelbourn?

Reply:
I like forcing people to use named parameters. `@include ...(solid, 3px, 4px)` is not as easy to read as `@include ...((border-style: solid, border-width: 3px, border-radius: 4px))`.

Reply:
Agreed, I prefer the map

Reply:
Okay, map it is. Resolving.